src/util/Util.js

```diff
@@ -54,7 +54,7 @@ class Util {
   static fetchRecommendedShards(token, guildsPerShard = 1000) {
     return new Promise((resolve, reject) => {
       if (!token) throw new Error('A token must be provided.');
-      snekfetch.get(Constants.Endpoints.gateway.bot)
+      snekfetch.get(`${Constants.DefaultOptions.http.host}${Constants.Endpoints.gateway.bot}`)
```

Comment:
Shouldn't that be `${Constants.DefaultOptions.http.host}/api${Constants.Endpoints.gateway.bot}` ?